src/scores/continuous/standard_impl.py

```diff
@@ -211,6 +211,16 @@ def correlation(
     """
     Calculates the Pearson's correlation coefficient between two xarray DataArrays
 
+    .. math::
+        \\rho = \\frac{\\sum{(x_i - \\bar{x})(y_i - \\bar{y})}}{\\sqrt{\\sum{(x_i-\\bar{x})^2}\\sum{(y_i - \\bar{y})^2}}}
```

Comment:
it maybe good to subscript the sum with `\sum_{i}` seeing as the variables have indexing. Although in this particular case it's fairly obvious.

Reply:
If you do this, it would be good to add a subscript too. i.e., change all the sums to
`\\sum_{i=1}^{n}`

Reply:
![image](https://github.com/nci/scores/assets/134973832/e6a2a4f4-52c3-42ef-9a5e-1c64966cad68)